Node Status Monitor/Python v2.7/process_data.py

```diff
@@ -0,0 +1,13 @@
+# Function to convert the api response to desired formate
+def process_node_details(node_details):
+    node_objects = []
```

Comment:
Why split `process_node_details` into it's own file?

Reply:
we are generating new object from existing node_details for efficient reading in future and data processing

Reply:
I don't understand how splitting this method to its own file would be more efficient.

Reply:
modification done, please review again